tools/run_tests/xds_k8s_test_driver/framework/infrastructure/gcp/compute.py

```diff
@@ -17,15 +17,23 @@
 import logging
 from typing import Any, Dict, List, Optional, Set
 
+from absl import flags
 from googleapiclient import discovery
 import googleapiclient.errors
 
+from framework import xds_flags
```

Comment:
Ideally these wrappers around GCP APIs should not be using xds_flags. They have nothing to do with xds itself - and we may end up using them for non-xds related cases.
Let's parse this flag in `TrafficDirectorManager.__init__` , and passing it as an argument to `ComputeV1`.

I don't like the fact `TrafficDirectorManager` parses `xds_flags` too, but, at least `TrafficDirectorManager` already does it for another flag.

Reply:
done